Tools/_sunrise/changelog/actions_changelogs_since_last_run.py

```
@@ -422,6 +480,87 @@ def get_past_runs(sess: requests.Session, current_run: Any, page: int = 1) -> An
     return resp.json()
 
 
+def get_earliest_target_attempt(
+    sess: requests.Session,
+    github_repository: str,
+    github_run: str,
+    target_id: str,
+) -> Any:
```

Ruff (0.16.3)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `get_earliest_target_attempt`

(ANN401)